docs/reference/settings/security-settings.asciidoc

```diff
@@ -1204,6 +1203,257 @@ If this setting is used, then the Kerberos realm does not perform role mapping a
 instead loads the user from the listed realms.
 See {stack-ov}/realm-chains.html#authorization_realms[Delegating authorization to another realm]
 
+[[ref-oidc-settings]]
+[float]
+===== OpenID Connect realm settings
+
+In addition to the <<ref-realm-settings,settings that are valid for all realms>>, you
+can specify  the following settings:
+
+`op.name`::
+The name of the OpenID Connect Provider, used as an identifier.
```

Comment:
Can you please check if this is still required?
`OpenIdConnectProviderConfiguration#getProviderName` seems unused.

Reply:
You're right. I'll remove the setting/code/tests in a followup